monero-outreach-docs/translations/it/quickfacts/QuickFacts_it.md

```diff
@@ -0,0 +1,127 @@
+**Hash Rate della rete:**435.17 MH/s
+**Media Transazioni/Ora:**178 (media su 30 gg)
+**Core CPU attivi sulla Rete:**14,499,178
+**Monero in Circolazione:**16,570,126 XMR (approssimati)
+**Capitalizzazione di Mercato:**$1,503,928,296 USD (~0.77% of della capitalizzazione totale delle criptovalute)
+**Ricompensa di blocco corrente:**3.5984 XMR
+**Tempo di blocco medio:**2 minuti
+
+_La ricompensa diminuirà gradualmente fino alla fine di maggio 2022, quando saranno in circolazione 18,132 milioni di XMR; a questo punto resterà una ricompensa di blocco di 0,6XMR._
+
+_Con un'"emissione di coda" di 0,6XMR/blocco, nel 2040 i Moneroj in circolazione saranno della stessa quantità di Bitcoin (circa 21 milioni)._
+### FUNZIONALITÀ IN FASE DI SVILUPPO
+
+Nonostante Monero sia già disponibile ed utilizzato in tutto il mondo, la comunità di sviluppatori ha obiettivi emozionanti volti a migliorare le funzionalità legate alla privacy, alla sicurezza e all'usabilità di Monero e delle criptovalute in generale. Eccone alcune che stanno per arrivare:
+
+**Kovri:** si tratta di un sostanziale miglioramento della privacy che aggiunge un quarto strato di privacy alle transazioni Monero. Kovri usa la **[garlic-encryption](https://getmonero.org/it/resources/moneropedia/garlic-encryption.html)** (si pensi a Tor) per mascherare l'indirizzo IP di mittenti e dei destinatari che utilizzano la rete. Ogni strato di una transazione Monero, il mittente, il destinatario, l'ammontare della transazione e (con Kovri) la colonna portante della transazione verranno mantenuti privati e sicuri.
+
+**Portafogli Hardware:** il portafoglio hardware Trezor T ha aggiunto Monero al set di **[monete supportate](https://trezor.io/coins/#XMR)**. Il **[portafoglio hardware Ledger](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** è già disponibile attraverso il portafoglio GUI Monero ufficiale. La comunità Monero ha finanziato un team di lavoro che sta realizzando **[Kastelo, un portafoglio hardware open source](http://kastelo.org/)**, basato sull'ethos Monero. Tutte queste opzioni hanno il totale supporto Monero.
+
+**Kasisto:** questa **[soluzione POS](https://github.com/amiuhle/kasisto)** è progettata per facilitare le vendite per commercianti e aziende.
+
+
+## ULTERIORI INFORMAZIONI
+
+**_[getmonero.org (Sito Ufficiale)](https://getmonero.org/it/)_**
+**_[monero.how](https://www.monero.how/)_**
+**_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+**_[Guide to Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+**_[Monero In-Depth Technical Intro](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+**_[Zero to Monero](https://www.getmonero.org/it/library/Zero-to-Monero-1-0-0.pdf)_**
+
+**_[Scams to Avoid](https://www.reddit.com/r/Monero/wiki/avoid)_**
+**_[Monero FAQ](https://ww.getmonero.org/it/get-started/faq/)_**
```

Comment:
There has been something curious happening when you make the search of the getmonero.org site, it shows only with two 'w' https://ww.get... instead of what should normally appear with three 'w'...https://www.getmon...

Rehrar told me that is something that has been happening but I guess he doesn't know exactly why.

https://www.getmonero.org/it/get-started/faq/